src/patternfly/components/Breadcrumb/breadcrumb.scss

```diff
@@ -0,0 +1,125 @@
+@import "../../patternfly-utilities";
+
+.pf-c-breadcrumb {
+  --pf-c-breadcrumb--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-breadcrumb--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-breadcrumb--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-breadcrumb--PaddingLeft: var(--pf-global--spacer--xl);
+
+  // Breadcrumb Item
+  --pf-c-breadcrumb__item--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-breadcrumb__item--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-breadcrumb__item--LineHeight: var(--pf-global--LineHeight--sm);
+  --pf-c-breadcrumb__item--Color: var(--pf-global--Color--300);
+  --pf-c-breadcrumb__item--hover--Color: var(--pf-global--Color--300);
+
+  // Breadcrumb Link
+  --pf-c-breadcrumb__link--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-breadcrumb__link--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-breadcrumb__link--LineHeight: var(--pf-global--LineHeight--sm);
+  --pf-c-breadcrumb__link--Color: var(--pf-global--link--Color);
+  --pf-c-breadcrumb__link--hover--Color: var(--pf-global--link--Color);
+  --pf-c-breadcrumb__link--hover--TextDecoration: var(--pf-global--link--TextDecoration--hover);
+
+  // Breadcrumb Divider
+  --pf-c-breadcrumb__divider--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-breadcrumb__divider--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-breadcrumb__divider--FontFamily: "Font Awesome 5 Free";
+  --pf-c-breadcrumb__divider--Color: var(--pf-global--Color--300);
+  --pf-c-breadcrumb__divider--Content: "\f105";
```

Comment:
I wonder if we should have a global var for this as well? I think we are using it in other places right?

Reply:
I think so, but since this is the only place using a FA string like this currently, I don't think it's necessary for this PR. I mentioned it in https://github.com/patternfly/patternfly-next/issues/830, maybe we can address it there and update this component accordingly?